data/card_catalogue.json

```diff
@@ -403,13 +403,13 @@
           }
         },
         {
-          "title": "Monk Mode",
+          "title": "Bashbolster",
           "upgrade": "",
           "tier": "green",
           "details": {
             "rules": "Unarmed attacks deal double damage until you play your next card.",
```

Comment:
We should change `unarmed` into `melee`. Players can't really be armed

Reply:
Sounds good, after I'm done with this I'll open an issue about reviewing the language in the catalogue - along with this there are some other semantic inconsitencies (i.e. "square" and "tile") I'd like to tidy up just for the sake of polish.